app/javascript/src/hyacinth_v1/components/projects/publish_targets/PublishTargetsEdit.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { startCase } from 'lodash';
+import produce from 'immer';
+
+import TabHeading from '../../shared/tabs/TabHeading';
+import { PublishTargetsForm } from './PublishTargetsForm';
+import { projects } from '../../../utils/hyacinthApi';
+import ProjectInterface from '../ProjectInterface';
+
+export default class PublishTargetsEdit extends React.Component {
```

Comment:
We're trying to go with functional components for all new components.  Why go class-based for this one?  Is there something that didn't work in a functional context?

Reply:
Nope, just expediency in implementation - the closest analog was a class-based component.

Reply:
Converted to functional component.